### ISSUE TITLE
Preview doesn't work if directory name has spaces

### ISSUE THREAD

**Issue by williamhCode** [nvim-telescope/telescope.nvim#2103]
### Description

Preview doesn't work if directory name has spaces. I was really confused at first and after opening several directories with `vim .`, I found that the directory names with spaces makes the preview disappear. I don't know if this is an macOS only issue. 

Note: I use the fzf extension, but tried it without and still didn't work.

### Neovim version

```markdown
NVIM v0.8.0-dev+528-g3b1423bfa
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macos 12.4

### Telescope version / branch / rev

telescope 0.1.0

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - WARNING: fd: not found. Install [sharkdp/fd](https://github.com/sharkdp/fd) for extended capabilities

## ===== Installed extensions =====

## Telescope Extension: `fzf`
  - INFO: No healthcheck provided
```

### Steps to reproduce

```zsh
mkdir "Test Dir"
cd "Test Dir"
nvim test.txt
```
type some stuff into `test.txt`, `:wq`,
```zsh
nvim .
```
activate telescope find files.

### Expected behavior

Something should show in the preview.

### Actual behavior

Nothing shows in the preview:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/83525937/181072827-91eebef1-7dcd-4e6d-8971-6f860a7049a4.png">

If I change `Test Dir` to `Test-Dir`:
```zsh
cd ..
mv "./Test Dir/" Test-Dir
cd Test-Dir
nvim .
```
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/83525937/181073507-d34c8dae-3507-4484-9318-88e34d9fe01b.png">

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('fzf')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by Conni2461** [comment]
Already fixed in 0.1.x branch Was a regression in 0.1.0

**Comment by williamhCode** [comment]
Switching to 0.1.x works thx. Also what does the x in the branch name mean?
